app/views/drinks/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>Edit Drink</h1>
+
+<% if @drink.errors.any? %>
+<ul>
+  <% @drink.errors.full_messages.each do |message| %>
+  <li> <%= message %> </li>
+  <% end %>
+</ul>
+<% end %>
+
+
+<%= form_for @drink do |f| %>
+  <div>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+  </div>
+  <div>
+    <%= f.label :description %>
+    <%= f.text_area :description %>
+  </div>
+  <% if @drink.alcoholic? %>
+    <p>Liquors</p>
+    <div>
+      <%= f.collection_check_boxes :liquor_ids,
+      @liquors,
+      :id,
+      :name
+      %>
+    </div>
```

Comment:
Would it be appropriate to apprise the user who is editing a non-alcoholic drink that the drink is in fact non-alcoholic? Do you do that somewhere else?